webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx

```diff
@@ -84,8 +81,10 @@ const ExperimentSingleTrialTabs: React.FC<Props> = (
         { id: experiment.id, limit: 2 },
         { signal: canceler.signal },
       );
-      if (expTrials.trials[0]) {
-        setFirstTrialId(expTrials.trials[0].id);
+      const firstTrial = expTrials.trials[0];
+      if (firstTrial) {
```

Comment:
nit: explicitly check against undefined? 0 could be a trial id sometime although it'd not be a valid item in the trials list now. feel free to ignore

Reply:
not 100% sure but I don't it's possible for `expTrials.trials` to be something other than an array, and if [0] exists, it is going to be an object instead of just a trial id or something to that effect.